README.md

```diff
@@ -1 +1,121 @@
-# Food-Recipe
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [Blog-App ](#blog-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Blog-App <a name="about-project"></a>
+
+**Recipe-app** The Recipe app keeps track of all your recipes, ingredients, and inventory
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Developed using Ruby on Rails.
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **Database that contains information about users, likes and posts**
+## 💻 Getting Started <a name="getting-started"></a>
+> To get a local copy up and running, follow these steps.
+- install gems bundle i
+- setup database rails db:setup 
+- run rails server rails s.
+- rails c
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install Ruby
+- Install PostgreSQL
+- Instal Rails gem
+
+### Setup
+```
+- To clone the repository run this command:
+
+ `git clone https://github.com/asnganje/Food-Recipe.git`
+
+
+
+- To access the cloned directory run:
+
+  `cd blog_app`
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update the above command to match the correct name of your project as the current is wrong. Good: `cd Food-Recipe`,  Bad: `cd blog_app`